engines/config-query-sparql/config/http/actors-wayback.json

```diff
@@ -0,0 +1,19 @@
+{
+  "@context": [
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/runner/^2.0.0/components/context.jsonld",
+
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-http-wayback/^2.0.0/components/context.jsonld",
+
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/bus-http/^2.0.0/components/context.jsonld"
+  ],
+  "@id": "urn:comunica:default:Runner",
+  "@type": "Runner",
+  "actors": [
+    {
+      "@id": "urn:comunica:default:http/actors#main",
```

Comment:
```suggestion
      "@id": "urn:comunica:default:http/actors#wayback",
```

Reply:
This change is by design. By default actor the `WayBack` actor is the only one subscribed to `#main` and the rest are subscribed to `#no-fallback` so that we can always do the wayback attempt on 404ing http requests in the default configuration.

Reply:
I suppose you're talking about the mediator or bus id?
But I was talking about the actor id here, for which we follow this naming strategy: https://github.com/comunica/comunica/tree/master/engines/config-query-sparql/config#iri-strategy

Reply:
I see you've already merged the PR, could you apply this fix in a separate PR?